bugbear.py

```diff
@@ -2380,6 +2391,9 @@ def visit_Lambda(self, node) -> None:
         "B909 editing a loop's mutable iterable often leads to unexpected results/bugs"
     )
 )
+B910 = Error(
+    message="B910 use Counter() instead of defaultdict(int) to avoid memory leaks"
```

Comment:
I'd change "memory leaks" to "memory usage", although that is perhaps a matter of perspective.

Reply:
Right. Fixed.